ytdlp_handler.py

```diff
@@ -39,11 +40,28 @@ def video_dl(values: Dict) -> None:
     with yt_dlp.YoutubeDL(ydl_opts) as ydl:
         infos_ydl = ydl.extract_info(values["url"])
     DL_PROGRESS_WINDOW.close()
+
+    if infos_ydl["_type"] == "playlist":
+        for infos_ydl_entry in infos_ydl["entries"]:
+            post_download(values, ydl, infos_ydl_entry)
+    else:
+        post_download(values, ydl, infos_ydl)
+
+
+def post_download(values: Dict, ydl, infos_ydl):
+    """
+    Execute all needed processes after a youtube video download :
+    - Execute not AudioOnly process
+    - Move output file(s)
+    """
     ext = "mp3" if values["AudioOnly"] else infos_ydl["ext"]
     full_path = os.path.splitext(ydl.prepare_filename(infos_ydl))[0] + "." + ext
     if not values["AudioOnly"]:
         post_process_dl(full_path, values["TargetCodec"])
 
+    # Move the output file to the user specified output directory
+    Path("{}".format(full_path)).rename("{}/{}".format(values["path"], full_path))
```

Comment:
Thanks for remarks !

I fix the first 3 points. The fourth is because it seems the output folder feature doesn't work.

In my dev environment and release (v0.0.5) the output files doesn't been moved after the download. There always in the same directory of the executable (video-dl.exe).

This is my configuration :

Installation directory : 

C:\Users\Dany\AppData\Local\Programs\Video-dl

![image](https://user-images.githubusercontent.com/28624619/147869880-085e27ae-1388-4cd7-bd52-3d3f96b95364.png)

The problem is the same if the executable and the output folder are not on the same drive (e.g : executable on C: and output in D:)